f2/src/utils/regex.js

```diff
@@ -1,6 +1,6 @@
 export const regexDef = () => {
   return {
     phoneRegExp: /^(\+?\d{0,4})?\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{3}\)?)\s?-?\s?(\(?\d{4}\)?)?$/,
-    postalCodeRegex: /^([A-Za-z]\d[A-Za-z][-]?\d[A-Za-z]\d)/,
+    postalCodeRegex: /^[ABCEGHJKLMNPRSTVXY]\d[ABCEGHJKLMNPRSTVWXYZ]( )?\d[ABCEGHJKLMNPRSTVWXYZ]\d$/i,
```

Comment:
Why did we need to change the postal code Regex?

Reply:
> Why did we need to change the postal code Regex?
I input post code  "z9z9z9" , the error message didn't show, that's why I use the previous postalCodeRegex,